src/database.cpp

```diff
@@ -4,14 +4,69 @@
 
 #include <boost/archive/binary_iarchive.hpp>
 #include <boost/archive/binary_oarchive.hpp>
+#include <filesystem>
 #include <silo/common/PerfEvent.hpp>
 #include <syncstream>
 #include <silo/common/hashing.h>
 #include <silo/common/istream_wrapper.h>
 #include <silo/database.h>
+#include <silo/prepare_dataset.h>
+#include <silo/preprocessing_config.h>
 #include <tbb/blocked_range.h>
 #include <tbb/parallel_for_each.h>
 
+const std::string REFERENCE_GENOME_FILENAME = "reference_genome.txt";
+const std::string PANGO_ALIAS_FILENAME = "pango_alias.txt";
+
+std::vector<std::string> silo::Database::initGlobalReference(const std::string& working_directory) {
+   std::ifstream reference_file(working_directory + REFERENCE_GENOME_FILENAME);
+   if (!reference_file) {
+      std::cerr << "Expected file " << working_directory << REFERENCE_GENOME_FILENAME << "." << std::endl;
```

Comment:
Let's get rid of the `std::cerr` here and in `initAliasKey`, while we're on it. Also let's improve the exception messages a bit to disclose more details. I imagine something like "Global reference genome file $file does not exist".